lms/djangoapps/verify_student/management/commands/approve_id_verifications.py

```diff
@@ -0,0 +1,121 @@
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'user_ids_file',
+            action='store',
+            help='Path of the file to read user IDs from.',
+            type=str,
+        )
+
+        parser.add_argument(
+            '--batch-size',
+            default=10000,
+            help='Maximum records to write in one query.',
+            type=int,
+        )
+
+    def handle(self, *args, **options):
+        user_ids_file = options['user_ids_file']
+        batch_size = options['batch_size']
+
+        if user_ids_file:
+            if not os.path.exists(user_ids_file):
+                raise CommandError('Pass the correct absolute path to user ID file as the first positional argument.')
+
+        total_users, failed_user_ids = self._approve_verifications_from_file(user_ids_file, batch_size)
+
+        if failed_user_ids:
+            log.error('Completed ID verification approvals. {} of {} failed.'.format(
+                len(failed_user_ids),
+                total_users,
+            ))
+            log.error(f'Failed user IDs:{pformat(failed_user_ids)}')
+        else:
+            log.info(f'Successfully approved ID verification attempts for {total_users} user IDs.')
+
+    def _approve_verifications_from_file(self, user_ids_file, batch_size=None):
+        """
+        Manually approve ID verification attempts for the user provided in the user IDs file.
+
+        Arguments:
+            user_ids_file (str): path of the file containing user ids.
+            batch_size (int): limits the number of verifications written to db at once
+
+        Returns:
+            (total_users, failed_user_ids): a tuple containing count of users processed and a list containing
+             user IDs whose verifications could not be processed.
+        """
+        with open(user_ids_file) as file_handler:
+            user_ids = [line.rstrip() for line in file_handler]
+            total_users = len(user_ids)
+
+        log.info(f'Attempting to manually approve ID verification attempts for {total_users} users.')
+        failed_user_ids = []
+        for n in range(0, total_users, batch_size):
+            failed_user_ids.extend(self._approve_id_verifications(user_ids[n:n + batch_size]))
+
+        return total_users, failed_user_ids
+
+
+    def _approve_id_verifications(self, user_ids):
+        """
+        This command manually approves ID verification attempts for a provided set of learners whose ID verification
+        attempt is in the submitted state. 
+
+        Arguments:
+            user_ids (list): user IDs of the users whose ID verification attempt should be manually approved
+
+        Returns:
+            failed_user_ids: list of user IDs for which a manual approval was not performed
+        """
+        users = User.objects.filter(id__in=user_ids)
+        existing_id_verifications = SoftwareSecurePhotoVerification.objects.filter(
```

Comment:
A user may have more than one `SoftwareSecurePhotoVerification` attempt, although it should be rare. I believe it's safe to approve them all.